### PR TITLE
Feature/conversion prep

### PR DESCRIPTION
**Pull request by tgurbich**
- Modified the final GFF file to work with converters.
- Added a script to generate a GFF file for conversion
- Fixed a bug in RNA output processing

### CODE REVIEW

File: bin/annotate_gff.py [hunk 1]
```diff
@@ -719,6 +722,163 @@ def get_ncrnas(ncrnas_file):
     return ncrnas
 
 
+def prepare_rna_gff_fields(cols):
+    if cols[1] in ["LSU_rRNA_bacteria", "SSU_rRNA_bacteria", "5S_rRNA"]:
+        rna_feature_name = "rRNA"
+    else:
+        rna_feature_name = "ncRNA"
```
Comment (mberacochea) [inline bin/annotate_gff.py:729]:
```suggestion
    rna_feature_name = "ncRNA"
    if cols[1] in ["LSU_rRNA_bacteria", "SSU_rRNA_bacteria", "5S_rRNA"]:
        rna_feature_name = "rRNA"
```

File: bin/prepare_gff_for_conversion.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        re.split(r"(?<!\\)=", item) for item in re.split(r"(?<!\\);", columns[8])
+    )
+    # Move all GO terms into Dbxref and deduplicate if needed
+    attributes_dict = collect_go_terms(attributes_dict)
+
+    # Move Pfam, Rfam and InterPro into Dbxref
+    terms_to_move = ["pfam", "rfam", "interpro"]
+    attributes_dict = move_to_dbxref(attributes_dict, terms_to_move)
+
+    attributes_to_move_to_note = {
```
Comment (mberacochea) [inline bin/prepare_gff_for_conversion.py:53]: Why do you change the names?

**Review by mberacochea — APPROVED** [review]
I gave it a quick read, it looks alright. In the new year we need to add some unit test and maybe a more in depth review.

File: bin/prepare_gff_for_conversion.py (hunk 2 above)
Comment (tgurbich) [reply bin/prepare_gff_for_conversion.py:53]: That's the convention for these names, they are defined in Dbxref. If not part of Dbxref, they need to be lowercase.

**Pull request merged**